examples/translate_rectangle.ml

```diff
@@ -0,0 +1,18 @@
+open Graphics
+
+let translate_rectangle dx dy rectangle =
+  match rectangle with
+  | Rectangle r -> Rectangle { r with c = { x = r.c.x + dx; y = r.c.y + dy }; }
+
+let () =
+  init ();
+  set_color black;
+
+  (* Create rectangle transform *)
+  let r1 = rectangle 200 100 in
+  let r2 = Rectangle_transform.translate_rectangle 100 0 r1 in
```

Comment:
Where is the `Rectangle_transform` module created?

Reply:
Oh, I initially created files for the transform modules and later on took them out and forgot to edit the code. I'll adjust them now. Thank you